packages/client/hmi-client/src/components/model/petrinet/tera-initial-table.vue

```diff
@@ -122,7 +122,7 @@ ul {
 	flex-grow: 1;
 	padding-left: var(--gap-1);
 
-	li {
+	& li {
```

Comment:
That's interesting that you need to add the ampersand 🤔

Reply:
I think it still may work but the colouring is off in my editor. From what I remember having the & is the "right way" to do it